### PR TITLE
Build new schemes

### PR DESCRIPTION
**Pull request by jocelynthode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,15 +61,17 @@ base16-black-metal-marduk
...
 base16-black-metal-venom
-base16-black-metal
 base16-blueforest
 base16-blueish
 base16-brewer
 base16-bright
 base16-brogrammer
-base16-brushtrees-dark
 base16-brushtrees
-base16-catppuccin
```
Comment (RRethy) [inline README.md:71]: Add this back.

**Review by RRethy — APPROVED** [review]
LGTM I can merge after the re-add of the catppuccin theme. I don't want to remove any themes since it would break a users config.

**Comment by jocelynthode** [comment]
@RRethy I just ran the script from the repo. Catppuccin was split into multiple subthemes. So if we want to restore this one we'll have to maintain it manually as it's not there in the main repo anymore.

**Comment by bezhermoso** [comment]
Hi, I think this can be closed. This PR covers these changes already: https://github.com/RRethy/nvim-base16/pull/75

**Pull request closed without merging**